backend/svirt.pm

```diff
@@ -36,6 +42,9 @@ sub new {
 sub do_start_vm {
     my ($self) = @_;
 
+    my $vars = \%bmwqemu::vars;
+    $vars->{NUMDISKS} ||= 1;    # this is what's supported now
```

Comment:
I don't get why the comment. Do you really mean just to filter out diskless systems?